src/components/LoansByCategory/FeaturedHeroLoanWrapper.vue

```diff
@@ -65,12 +65,19 @@ export default {
 			type: Boolean,
 			default: false,
 		},
+		favoriteSectorId: {
+			type: String,
+			default: null
+		},
+		featuredSectorExpVersion: {
+			type: String,
+			default: null
+		}
 	},
 	data() {
 		return {
 			experimentData: {},
-			featuredCategoryIds,
-			initialLoanCount,
+			featuredCategoryIds: [56],
```

Comment:
These were never getting updated to have a value in the code below...So we'll just go with the hardcoded value for the default loan channel and the global limit.